Vic2ToHoI4/Source/Hoi4Outputter/DifficultySettings.h

```diff
@@ -0,0 +1,13 @@
+#ifndef DIFFICULTY_SETTINGS_OUTPUTTER
+#define DIFFICULTY_SETTINGS_OUTPUTTER
+
+#include "../Configuration.h"
+
+namespace HoI4
+{
+
+void outputDifficultySettings(std::string difficultySettings, const Configuration& theConfiguration);
```

Comment:
Take the great powers as a parameter instead of generating the text in HoI4::World.

Reply:
Done